gperftools/plan.sh

```diff
@@ -1,14 +1,26 @@
 pkg_name=gperftools
 pkg_origin=core
-pkg_version=2.6.90
+pkg_version=2.7
 pkg_description="Google Performance Tools"
 pkg_upstream_url=https://github.com/gperftools/gperftools
 pkg_license=('BSDv3')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source="https://github.com/gperftools/gperftools/releases/download/${pkg_name}-${pkg_version}/${pkg_name}-${pkg_version}.tar.gz"
-pkg_shasum=2f8ec5129701e6c68c75009db5303761b65b51867c63e49ec19b946930cfef1f
-pkg_deps=(core/glibc core/gcc-libs core/graphviz core/coreutils core/grep core/perl core/binutils)
-pkg_build_deps=(core/gcc core/make core/automake)
+pkg_shasum=1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9
+pkg_build_deps=(
```

Comment:
👍 for the readability on moving the deps onto single lines for readability.  I'm not keen on the alphabetizing (see https://github.com/habitat-sh/core-plans/pull/1786#discussion_r210015876 for details why) but I don't _think_ there will be any conflicts in this list.